.github/actions/publish-nuget/New-NuGetPackage.ps1

```diff
@@ -126,6 +100,41 @@ foreach ($project in $projects)
         continue
     }
 
+    $PackageValidationParameters = @(
+        "-p:EnablePackageValidation=$EnablePackageValidation"
+        "-p:PackageValidationBaselineVersion=$PackageValidationBaselineVersion"
+    )
+
+    # Download baseline version NuGet packages
+    if ($shouldDownloadBaseLinePackages)
+    {
+        Write-Output 'Creating temporary project for baseline NuGet package.'
+        dotnet new classlib -n TempProject
+        Push-Location TempProject
+
+        Write-Output 'Installing baseline version NuGet package.'
+        dotnet add TempProject.csproj package $project.BaseName --version $PackageValidationBaselineVersion
+
+        if ($LASTEXITCODE -ne 0)
+        {
+            Write-Output "Package version couldn't be added, thus package validation to baseline version won't be done."
```

Comment:
This seems fairly exceptional so I think it should appear on the workflow summary page too:
```suggestion
            Write-Output '::warning::Package version couldn't be added, thus package validation to baseline version won't be done.'
```

Also constant string literals should be bordered with single quotes, see [AvoidUsingDoubleQuotesForConstantString](https://learn.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/rules/avoidusingdoublequotesforconstantstring?view=ps-modules).